### ISSUE TITLE
Refactor "Settings" panel of Navigation Item block to use `ToolsPanel` instead of `PanelBody`

### ISSUE THREAD

**Issue by fabiankaegy** [WordPress/gutenberg#67947]
Part of #67813 

Convert the "Settings" panel of the Navigation Item block to use the `ToolsPanel` instead of `PanelBody`

**Comment by ellatrix** [comment]
Hey @fabiankaegy, why not create a single issue with a checklist? It's a bit overwhelming to have so many issues about this.

**Comment by fabiankaegy** [comment]
@ellatrix We started with that but quickly ran into the issue that with 5+ different contributors it was hard to keep track of who would work on which tasks which led to duplicate work that we had to close. That is when we decided to create the child tasks so it would be easier to identify who works on what. 

Sorry that it adds an overwhelming amount of stuff to the issues tab. 

I had hoped the new GitHub Sub Issue feature would actually make it so they are hidden from the main issue list. (would be much nicer!)

If you think it's better I can go close the issues and go back to a list?
